lib/controllers/skill-code-controller/code-builder.js

```diff
@@ -100,7 +87,7 @@ class CodeBuilder {
         // 2.infer from the src code to decide the build flow
         this.buildFlow = this._inferCodeBuildFlow();
         this.packageBuilder = this.buildFlow === ZIP_ONLY_BUILD_FLOW ? null
-            : path.join(__dirname, '..', '..', 'builtins', 'build-flows', this.buildFlow, this.osType === 'WINDOWS' ? 'build.ps1' : 'build.sh');
+            : path.join(__dirname, '..', '..', 'builtins', 'build-flows', this.buildFlow, this.isWindows ? 'build.ps1' : 'build.sh');
```

Comment:
In your current usage, I think this line can be removed.

However, I would suggest we still treat language specific built-in script into separate files... (instead of listing those methods in one file) this is because it's more scalable since the supported built-in languages may grow. Though it can be short file, it still is well-boundary. Thus I think we should still keep using .packageBuilder to point to the execution logic.